### ISSUE TITLE
(Suggestion) a button that makes the chat box disappear 

### ISSUE THREAD

**Issue by GeekyT69** [sm64js/sm64js#19]

**Comment by sm64js** [comment]
good idea, will add

**Comment by AgentXLP** [comment]
What's the benefit of hiding the chat? I don't understand

**Comment by Spasmworld** [comment]
@Agent-11 it would be nice to record stuff without it.

**Comment by AgentXLP** [comment]
> @Agent-11 it would be nice to record stuff without it.

I see

**Comment by g3-geo** [comment]
yeah and it kinda gets annoying to see people arguing/fighting in chat, whereas the point of sm64js is to just play online and not twitter so enabling/disabling chat would be cool
